File: C-Sharp/ClassLibraryAT/Smsshortcode.cs

    // Sending Messages using sender id/short code
    using System;
    class Smsshortcode
    {
        static public void _Smsshortcode (){
            
            string username = "MyAfricasTalkingUsername";
            string apiKey   = "MyAfricasTalkingAPIKey"; 
            
            string recipients = "+254711XXXYYY,+254733YYYZZZ";
            
            string message = "I'm a lumberjack and its ok, I sleep all night and I work all day";
            
            // Specify your AfricasTalking shortCode or sender id
            string from = "shortCode or senderId";
            
            AfricasTalkingGateway gateway = new AfricasTalkingGateway (username, apiKey);
            
            try {
                
                dynamic results = gateway.sendMessage (recipients, message, from);
                
                foreach( dynamic result  in results){
                    Console.Write((string)result["number"] + ",");
                    Console.Write((string)result["status"] + ",");
                    Console.Write((string)result["messageId"] + ",");
                    Console.WriteLine((string)result["cost"]);
                }
            } catch (AfricasTalkingGatewayException e) {
                Console.WriteLine ("Encountered an error: " + e.Message);        
            }
            Console.Read();
        }
    }

File: VB.Net/AfricasTalkingGateway/Smsshortcode.vb
	' Sending Messages using sender id/short code
	Imports System
	Imports System.Collections
	Friend Class Smsshortcode
    Public Shared Sub _Smsshortcode()

        Dim username As String = "MyAfricasTalkingUsername"
        Dim apiKey As String = "MyAfricasTalkingAPIKey"

        Dim recipients As String = "+254711XXXYYY,+254733YYYZZZ"

        Dim message As String = "I'm a lumberjack and its ok, I sleep all night and I work all day"

        ' Specify your AfricasTalking shortCode or sender id
        Dim from As String = "shortCode or senderId"

        Dim gateway As New AfricasTalkingGateway(username, apiKey)

        Try

            Dim results As Object = gateway.sendMessage(recipients, message, from)

            For Each result As Object In results
                Console.Write(CStr(result("number")) & ",")
                Console.Write(CStr(result("status")) & ",")
                Console.Write(CStr(result("messageId")) & ",")
                Console.WriteLine(CStr(result("cost")))
            Next result
        Catch e As AfricasTalkingGatewayException
            Console.WriteLine("Encountered an error: " & e.Message)
        End Try
        Console.Read()
    End Sub
End Class